components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpOperations.java

```diff
@@ -1074,4 +1097,64 @@ public synchronized boolean sendSiteCommand(String command) throws GenericFileOp
         // is not implemented
         return true;
     }
+
+    /*
+     * adapted from com.jcraft.jsch.Util.createSocket(String, int, int)
+     *
+     * added possibility to specify the address of the local network interface, against the
+     * connection should bind
+     */
+    static Socket createSocketUtil(final String host, final int port, final String bindAddress, final int timeout) {
```

Comment:
what you do is reasonably good as the class and method has default access modifers.